diff --git a/resources/views/layouts/default.blade.php b/resources/views/layouts/default.blade.php
index a8d5546e76..e2b03184cc 100644
--- a/resources/views/layouts/default.blade.php
+++ b/resources/views/layouts/default.blade.php
@@ -156,14 +156,14 @@
                                     </a>
                                 </li>
                             @endcan
-                            @can('view', \App\Models\Component::class)
-                                <li aria-hidden="true"{!! (request()->is('components*') ? ' class="active" aria-current="page"' : '') !!}>
+                            {{-- @can('view', \App\Models\Component::class)
+                                <li aria-hidden="true"{!! (Request::is('components*') ? ' class="active"' : '') !!}>
                                     <a href="{{ route('components.index') }}" {{$snipeSettings->shortcuts_enabled == 1 ? "accesskey=5" : ''}} tabindex="-1" data-tooltip="true" data-placement="bottom" data-title="{{ trans('general.components') }}">
                                         <x-icon type="components" class="fa-fw" />
                                         <span class="sr-only">{{ trans('general.components') }}</span>
                                     </a>
                                 </li>
-                            @endcan
+                            @endcan --}}
 
                             @can('index', \App\Models\User::class)
                                 <li aria-hidden="true"{!! (request()->is('users*') ? ' class="active" aria-current="page"' : '') !!}>
@@ -233,14 +233,14 @@
                                                 </a>
                                             </li>
                                         @endcan
-                                        @can('create', \App\Models\Component::class)
-                                            <li {!! (request()->is('components/create') ? 'class="active"' : '') !!}>
+                                        {{-- @can('create', \App\Models\Component::class)
+                                            <li {!! (Request::is('components/create') ? 'class="active"' : '') !!}>
                                                 <a href="{{ route('components.create') }}" tabindex="-1">
                                                     <x-icon type="components" class="fa-fw" />
                                                     {{ trans('general.component') }}
                                                 </a>
                                             </li>
-                                        @endcan
+                                        @endcan --}}
                                         @can('create', \App\Models\User::class)
                                             <li {!! (request()->is('users/create') ? 'class="active"' : '') !!}>
                                                 <a href="{{ route('users.create') }}" tabindex="-1">
@@ -596,14 +596,14 @@
                                 </a>
                             </li>
                         @endcan
-                        @can('view', \App\Models\Component::class)
-                            <li id="components-sidenav-option"{!! (request()->is('components*') ? ' class="active" aria-current="page"' : '') !!}>
+                        {{-- @can('view', \App\Models\Component::class)
+                            <li{!! (Request::is('components*') ? ' class="active"' : '') !!}>
                                 <a href="{{ route('components.index') }}">
                                     <x-icon type="components" class="fa-fw" />
                                     <span>{{ trans('general.components') }}</span>
                                 </a>
                             </li>
-                        @endcan
+                        @endcan --}}
                         @can('view', \App\Models\PredefinedKit::class)
                             <li id="kits-sidenav-option"{!! (request()->is('kits') ? ' class="active" aria-current="page"' : '') !!}>
                                 <a href="{{ route('kits.index') }}">
@@ -962,23 +962,9 @@
             </div><!-- /.content-wrapper -->
             <footer class="main-footer hidden-print" style="display:grid;flex-direction:column;">
 
-                <div class="hidden-xs pull-left">
-                    <div class="pull-left footer-links">
-                         {!! trans('general.footer_credit') !!}
-
-                        <a target="_blank" href="https://bsky.app/profile/snipeitapp.com" rel="noopener" data-tooltip="true" data-title="Join us on Bluesky">
-                            <i class="fa-brands fa-square-bluesky fa-fw"></i>
-                        </a>
-                        <a target="_blank" href="https://github.com/grokability/snipe-it/" rel="noopener" data-tooltip="true" data-title="Join us on Github">
-                            <i class="fa-brands fa-square-github fa-fw"></i>
-                        </a>
-                        <a target="_blank" href="https://hachyderm.io/@grokability" rel="noopener" data-tooltip="true" data-title="Join us on Mastodon">
-                            <i class="fa-brands fa-mastodon fa-fw"></i>
-                        </a>
-                        <a target="_blank" href="https://discord.gg/yZFtShAcKk" rel="noopener" data-tooltip="true" data-title="Join us on Discord">
-                            <i class="fa-brands fa-discord fa-fw"></i>
-                        </a>
-
+                <div class="1hidden-xs pull-left">
+                    <div class="pull-left" >
+                       <a target="_blank" href="http://walit.walcourt.local" rel="noopener">WAL-IT</a> Gestion des actifs </a>.
                     </div>
                     <div class="pull-right">
                     @if ($snipeSettings->version_footer!='off')
@@ -988,13 +974,7 @@
                         @endif
                     @endif
 
-                    @if (isset($user) && ($user->isSuperUser()) && (app()->environment('local')))
-                       <a href="{{ url('telescope') }}" class="label label-default" rel="noopener">Open Telescope</a>
-                    @endif
-
-
-
-
+               
                     @if ($snipeSettings->support_footer!='off')
                         @if (($snipeSettings->support_footer=='on') || (($snipeSettings->support_footer=='admin') && (Auth::user()->isSuperUser()=='1')))
                             <a target="_blank" class="label label-default"
@@ -1009,7 +989,7 @@
                         <a target="_blank" class="label label-default" rel="noopener"
                            href="{{  $snipeSettings->privacy_policy_link }}"
                            target="_new">{{ trans('admin/settings/general.privacy_policy') }}</a>
-                    @endif
+                    @endif --}}
                     </div>
                     <br>
                     @if ($snipeSettings->footer_text!='')
@@ -1478,4 +1458,4 @@
         @include('partials.theme-mode-toggle')
 
         </body>
-</html>
+</html>
\ No newline at end of file
diff --git a/resources/views/partials/bootstrap-table.blade.php b/resources/views/partials/bootstrap-table.blade.php
index 2e1259f165..321a6eba4b 100644
--- a/resources/views/partials/bootstrap-table.blade.php
+++ b/resources/views/partials/bootstrap-table.blade.php
@@ -1594,36 +1594,6 @@
     });
     @endcan
 
-    @can('create', \App\Models\Component::class)
-    // Component table buttons
-    window.componentButtons = () => ({
-        btnAdd: {
-            text: '{{ trans('general.create') }}',
-            icon: 'fa fa-plus',
-            event () {
-                window.location.href = '{{ route('components.create') }}';
-            },
-            attributes: {
-                class: 'btn-warning',
-                title: '{{ trans('general.create') }}',
-                @if ($snipeSettings->shortcuts_enabled == 1)
-                accesskey: 'n'
-                @endif
-            }
-        },
-        btnExport: {
-            text: '{{ trans('general.custom_component_report') }}',
-            icon: 'fa-solid fa-file-csv',
-            event () {
-                window.location.href = '{{ route('reports.custom.component') }}';
-            },
-            attributes: {
-                title: '{{ trans('general.custom_component_report') }}',
-            }
-        },
-    });
-    @endcan
-
     @can('create', \App\Models\Consumable::class)
     // Consumable table buttons
     window.consumableButtons = () => ({
